mix/QVariableDefinition.cpp

```diff
@@ -53,3 +55,95 @@ QVariableDefinition* QVariableDefinitionList::val(int _idx)
 		return nullptr;
 	return m_def.at(_idx);
 }
+
+/*
+ * QIntType
+ */
+dev::bytes QIntType::encodeValue()
+{
+	dev::bigint i(value().toStdString());
+	if (i < 0)
+		i = i + dev::bigint("0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff") + 1;
+	std::ostringstream s;
+	s << std::hex << "0x" << i;
+	return padded(jsToBytes(s.str()), 32);
+}
+
+void QIntType::decodeValue(std::string const& _rawValue)
```

Comment:
so we need to encode to byte in order to call m_client.transact(). And we need to decode to string ... to use the return parameter. I don't get your point here.